content/manuals/engine/daemon/_index.md

```diff
@@ -98,24 +98,34 @@ The Docker daemon persists all data in a single directory. This tracks
 everything related to Docker, including containers, images, volumes, service
 definition, and secrets.
 
-By default this directory is:
+By default the daemon stores data in:
 
-- `/var/lib/docker` on Linux.
-- `C:\ProgramData\docker` on Windows.
+- `/var/lib/docker` on Linux
+- `C:\ProgramData\docker` on Windows
 
-You can configure the Docker daemon to use a different directory, using the
-`data-root` configuration option. For example:
+When using the [containerd image store](/manuals/engine/storage/containerd.md)
+(the default for Docker Engine 29.0 and later on fresh installations), image
+and container snapshots are stored in `/var/lib/containerd`. Other daemon data
+(volumes, configs, secrets) remains in `/var/lib/docker`.
+
+When using [classic storage drivers](/manuals/engine/storage/drivers/_index.md)
+like `overlay2` (the default for upgraded installations), all data is stored in
+`/var/lib/docker`.
+
+### Configure the data directory location
+
+You can configure the Docker daemon to use a different directory using the
+`data-root` configuration option:
```

Comment:
We should add a note that this doesn't affect things under `/var/lib/container`. For that the users needs to configure the system containerd instance via `/etc/containerd/config.toml`

Reply:
Good shout!